### PR TITLE
feat(icons): add sound on off icons

### PR DESCRIPTION
**Pull request by bobbonius**
# Description

Added `SoundOn` and `SoundOff` icons to use in our video player for the event header.

## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Go to icons, see how they look

## Screenshots

image

## To be notified

@TicketSwap/frontend

### CODE REVIEW

File: src/icons/SoundOff.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { SvgWrapper } from './SvgWrapper'
+
+export const SoundOff = (props: any) => (
+  <SvgWrapper a11yTitle="SoundCloud" {...props}>
```
Comment (RobVermeer) [inline src/icons/SoundOff.tsx]: Don't think the title is right here

File: src/icons/SoundOn.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { SvgWrapper } from './SvgWrapper'
+
+export const SoundOn = (props: any) => (
+  <SvgWrapper a11yTitle="SoundCloud" {...props}>
```
Comment (RobVermeer) [inline src/icons/SoundOn.tsx]: Don't think the title is right here

**Review by RobVermeer — APPROVED** [review]

**Review by glenngijsberts — APPROVED** [review]
👍

**Pull request merged**